applications/GeoMechanicsApplication/tests/line_interface_elements/Neumann_single_stage/MaterialParameters.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
We deliberately use the same material parameters for the Neumann and for the Dirichlet test. Maybe we should make that very explicit by refering to the same file. 
For the mdpa files the different conditions require different model parts. They could all be in the same file and then the projectparameter file decides what is actually used. However that is a step further than sharing the material parameters file. Maybe we discuss this with more people shortly. In the thermal tests we benefitted from less files.

Reply:
I put the material parameters in a common folder. Since there are indeed differences for the mdpa, I think it's better to keep them as copies for now. Since it's only the mesh, I also don't expect too many changes in these files because of new/different functionality in geomechanics.